packages/app/src/elements/import-dialog.ts

```diff
@@ -40,14 +97,126 @@ export class ImportDialog extends Dialog<File, void> {
                 ></pl-select>
 
                 <div class="small padded" ?hidden=${this._formatSelect && this._formatSelect.value !== imp.CSV.value}>
-                    ${$l(
-                        "IMPORTANT: Before importing, please make sure that your CSV data " +
-                            "is structured according to {0}'s specific requirements!",
-                        process.env.PL_APP_NAME!
-                    )}
-                    <a href="#" @click=${this._downloadCSVSampleFile}> ${$l("Download Sample File")} </a>
+                    ${$l("Choose the correct column names and types for each column below. If you are having trouble,")}
+                    <a href="#" @click=${this._downloadCSVSampleFile}> ${$l("Download the Sample File")} </a>.
```

Comment:
I don't really think the sample file makes sense here any more. Instead, there should be a link to the (yet to be written) manual page. So I guess just remove the second part for now, and I'll add the link to the manual page once it exists.